CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to the Modify Contact Admin Button project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.1.1] - 2022-03-07
+### Changed
+- PHP8 compatibility fixes (GR)
+
```

Comment:
Let's strike this part. We write ChangeLog as part of the release workflow. The data and sometimes even the version number are unknown until the moment of release. Pre-drafting the content interferes with the scripts we use to do the releases.

```suggestion
```